common/common/src/main/java/io/helidon/common/HelidonFeatures.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Don't forget to update all the copyrights to 2020

Reply:
I will only update copyrights of files changed in year 2020 - these changes were done last year, so 2019 should be the correct year, right?

Reply:
Depends on the git commit date.